massa-models/src/config/constants.rs

```diff
@@ -84,6 +84,21 @@ lazy_static::lazy_static! {
         .parse()
         .unwrap()
     };
+    /// node chain id (to avoid replay attacks)
+    pub static ref CHAINID: u64 = {
+        // MASSA..
+        // MASM (MainNet):           77658377
+        // MASB (BuildNet):          77658366
+        // MASS (SecureNet):         77658383
+        // MASS275 (SecureNet 27.5): 77658383505553
```

Comment:
maybe remove this one for now, it can create uncertainty on which chainid to use for a given version

Reply:
Removed